src/components/ManageSigners/NewSignerForm.tsx

```diff
@@ -39,20 +43,24 @@ function NewSignerForm(props: Props) {
             autoFocus={process.env.PLATFORM !== "ios"}
             error={!!props.errors.publicKey}
             label={props.errors.publicKey ? props.errors.publicKey.message : "Public Key or Stellar Address"}
-            placeholder="GABCDEFGHIJK... or alice*example.org"
+            placeholder={isSmallScreen ? "GABCDEFGHIJK..." : "GABCDEFGHIJK... or alice*example.org"}
```

Comment:
[Nitpick] How about replacing the three dots with a single `…` and make it `GABCD… or address` on small screens?